pom.xml

```diff
@@ -169,6 +169,19 @@
       <version>3.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <!-- TODO we should pull out the Google Cloud support to a new library -->
```

Comment:
@mattnworb Just curious. What did you have in mind for this?

Reply:
see these two comments:
https://github.com/spotify/dockerfile-maven/issues/54#issuecomment-319155527
https://github.com/spotify/dockerfile-maven/issues/54#issuecomment-319764523